### ISSUE TITLE
tryed ordering on pcbway but they need the exact part number 

### ISSUE THREAD

**Issue by Shiroe93** [xyzroe/ZigUSB#6]
hello i tried to order the pcb on pcbway but they need the exact part number and one item is out of stock
could you please help me answer the question from pcb way about your bom?
[BOM_ZigUSB_touch.csv](https://github.com/xyzroe/ZigUSB/files/14948582/BOM_ZigUSB_touch.csv)
do you have an stl model for the case?

**Comment by xyzroe** [comment]
No, sorry. 
Also no 😞

**Comment by Shiroe93** [comment]
could you suggest me how/where to buy the missing part so i can provide them myself to pcbway?

**Comment by xyzroe** [comment]
I think you could use google to find out that.

**Comment by xyzroe** [comment]
<img width="1141" alt="Screenshot 2024-04-12 at 2 28 26 PM" src="https://github.com/xyzroe/ZigUSB/assets/6440415/8d43e1df-6773-454b-8d59-054bee28cc68">

it wasn't so difficult

**Comment by Shiroe93** [comment]
Yes i find all of them except a substitute for the zigbee module that they don't have in stock (and i was not able to find in any other store) unfortunately i don't understand enough about the project to determine if an alternative is viable do you have any suggestion?
I'm sorry for not being clear about that

**Comment by maxandcheeses** [comment]
@xyzroe Would you happen to have an export of the pick and place file?

**Comment by xyzroe** [comment]
no. You should solder by hands.
